library/CHART-TEMPLATE/tests/cl-apikey_secret_test.yaml

```diff
@@ -0,0 +1,56 @@
+suite: test insightsKey secret
+templates:
+  - templates/example-insights-secret.yaml
```

Comment:
As discussed offline this file should check the `example-apiKey-secret.yaml` instead 📋

Reply:
Really good point. Some copy and paste that I forgot to change. Done in [#15a9b7b](https://github.com/newrelic/helm-charts/pull/1463/commits/15a9b7b7959af5198c2786fe11f58bec8a0f33a3)